core/resource.cpp

```diff
@@ -431,7 +432,14 @@ void ResourceCache::setup() {
 
 void ResourceCache::clear() {
 	if (resources.size()) {
-		ERR_PRINT("Resources Still in use at Exit!");
+		ERR_PRINT("Resources still in use at exit (run with --verbose for details).");
```

Comment:
what if it's already running in verbose? this should be inside the else right?
```c++
if (OS::get_singleton()->is_stdout_verbose()) {
   ...
} else {
    ERR_PRINT("Resources still in use at exit (run with --verbose for details).");
}
```

Reply:
This would force using `ERR_PRINT` with both branches with just this message as difference, I'm not sure it's worth the trouble. The hint is still valid when running with `--verbose`, it doesn't say that there will be *more* details than currently ;)